job-dsls/jobs/new_pr_jobs.groovy

```diff
@@ -81,7 +81,11 @@ def final REPO_CONFIGS = [
         "kie-wb-common"             : [
                 timeoutMins: 300,
                 label: "kie-rhel7 && kie-mem16g && gui-testing",
-                executionNumber : 25
+                executionNumber : 25,
+                artifactsToArchive: [
+                        "kie-wb-common-stunner-bpmn-kogito-runtime/target/screenshots/**",
+                        "kie-wb-common-dmn-webapp-kogito-runtime/target/screenshots/**"
```

Comment:
these modules are not on top level of the repository, maybe do just wild card here 
```suggestion
                        "**/target/screenshots/**"
```

Reply:
done

Reply:
feel free to merge